sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -259,32 +257,6 @@ def on_response(self, request, response):
                 _LOGGER.debug("Failed to log response: %s", repr(err))
 
 
-class StorageUserAgentPolicy(SansIOHTTPPolicy):
```

Comment:
can we just inherit UserAgentPolicy from azure-core so that we can avoid _user_agent.py in every SDK

Reply:
_user_agent.py is removed now. We will directly use the sdk_monkier from the azure core UApolicy.

Reply:
So you are working on that right?

Reply:
pushing changed shortly.

Reply:
fixed